source/js/index.js

```diff
@@ -0,0 +1,36 @@
+import(/* webpackChunkName: "sentry" */ '@sentry/browser').then(module => {
```

Comment:
Do we need Sentry for production?
We might need to get a dsn for ourselves too.

Reply:
The DSN being used here is the same one currently in use on the Doctrine website. I assume it's owned by a contributor. If not we can remove Sentry for now

Reply:
Ping @jwage 
Is this still an active dsn? Do we currently use Sentry?